test/Libraries/SystemTestServices/DispatcherUtil.cs

```diff
@@ -1,13 +1,8 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Security.Permissions;
-using System.Windows.Threading;
 using System.Threading;
-using NUnit.Framework;
+using System.Windows.Threading;
 
-namespace DynamoCoreWpfTests.Utility
+namespace SystemTestServices
```

Comment:
After the namespace change, because it is a public static class, I believe 3rd party view extensions can write UI tests with the current Nuget

Reply:
But this will break previous code. For example, try to run the existing test assemblies.
You can move the code and use a type forward attribute to do this safely, but you can't rename the type including its namespace.

Reply:
Good idea, I will take a look of using typeforwarding